### PR TITLE
feat: t3001

### PR DESCRIPTION
**Pull request by Abishek666**
Creating local node server for the web page

### CODE REVIEW

**Comment by saikrishnanp** [comment]
Why some of the files are in cjs and not js?

File: server.cjs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const http = require('http')
```
Comment (saikrishnanp) [inline server.cjs:1]: Can you add a script command to package json to start the node server?

File: server.cjs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const path = require('path')
```
Comment (saikrishnanp) [inline server.cjs:2]: After starting the server it doesn't show any intimation regarding server is started or not. 
Can you add a log statement like server started with the port number?
image

File: server.cjs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      break
+    case '.png':
+  }
+  if (req.url === '/all-city-timezones') {
+    allCityData = timezone.allTimeZones()
+    res.writeHead(200, { 'Content-Type': 'application/json' })
+    res.end(JSON.stringify(allCityData))
+  }
+
+  if (req.url === '/get-hourly-details' && req.method === 'POST') {
```
Comment (saikrishnanp) [inline server.cjs]: The endpoints can be stored in any const variable and reused

File: src/index.html [hunk 4]
```diff
@@ -28,7 +28,7 @@
         </div>
         <header class="header-img">
             <img alt="leftcloudimg" class="left-cloud" src="../Assets/HTML&CSS/GeneralImages&Icons/cloud-8121.png">
-            <img alt="lightning" class="lightning" src="../Assets/HTML&CSS/lightning/pngwing.com (3).png">
```
Comment (saikrishnanp) [inline src/index.html:31]: Why is the middle sections tiles not interacting properly on hovering? Also horizontal scroll seems to not work properly

File: server.cjs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    case '.css':
+      filePath = './src' + reqUrl
+      contentType = 'text/css'
+      break
+    case '.js':
+      filePath = './src' + reqUrl
+      contentType = 'text/javascript'
+      break
+    case '.svg':
+      contentType = 'image/svg+xml'
```
Comment (saikrishnanp) [inline server.cjs:26]: Is the `filePath` not needed for .svg file type?

File: server.cjs [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      break
+    case '.ico' :
+      filePath = '/Assets/HTML&CSS/Weather_Icons/cloudyIcon.svg'
+      contentType = 'image/svg+xml'
+      break
+    case '.png':
```
Comment (saikrishnanp) [inline server.cjs]: If .png file type is not needed let's not have a case for it.
Also let's add a default case.

**Review by saikrishnanp — CHANGES_REQUESTED** [review]

**Comment by Abishek666** [comment]
> Why some of the files are in cjs and not js?
since we given type as module in package.json, it will consider all  js files as module and we have to give extension as cjs to escape 
from eslint error because we are using require keyword

File: server.cjs (hunk 5 above)
Comment (Abishek666) [reply server.cjs:26]: no need Sai, it will automatically search in directory

File: src/index.html (hunk 4 above)
Comment (saikrishnanp) [reply src/index.html:31]: The horizontal scroll still doesn't seem to be working properly. Not by clicking on the scroll arrow button but by mouse scroll(hold shift + scroll).

**Review by saikrishnanp — APPROVED** [review]

File: src/index.html (hunk 4 above)
Comment (Abishek666) [reply src/index.html:31]: Its working for me Sai

**Pull request merged**